Scheduler:  add example for onOptionChanged and edit currentDate, currentView, views.name, views.type (#1517)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxScheduler/1 Configuration/currentDate.md

---
---
##### shortDescription
Specifies the current date.

---

> Users can change this value using the Date Navigator:

Reviewer: 1. I suggest you to add a link
2. using the Date Navigator -> in the `[Date Navigator](link)`

> Users can change this value in the [Date Navigator](/Documentation/Guide/Widgets/Scheduler/Date_Navigator/):

Reviewer 2: ```suggestion
Users can change the current date in the [Date Navigator](/Documentation/Guide/Widgets/Scheduler/Date_Navigator/):
```

![Scheduler Date Navigator](/images/UiWidgets/Scheduler_Date_Navigator.png)

> To subscribe to changes of the current date, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) option.

Reviewer: ```suggestion
To subscribe to changes of the current date, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) handler.
```

> To subscribe to changes for the current date, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) handler.

Reviewer 2: ```suggestion
To subscribe to changes of the current date, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) handler.
```
